src/Faker/Provider/bg_BG/Address.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Faker\Provider\bg_BG;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $buildingNumber = array('##', 'номер #');
+    protected static $cityPrefix = 'гр.';
+    protected static $citySuffix = array('град');
+    protected static $streetPrefix = array('улица', 'ул.');
+
+    protected static $postcode = "####";
+    protected static $country = array( 'България' );
+    protected static $cityFormats = array(
+        '{{cityPrefix}} {{lastName}}{{citySuffix}}',
```

Comment:
no, using last name for city names doesn't provide good results. Please use a list of 100-300 real city names, preferably the most populated ones.